lms/templates/discussion/_discussion_module.html

```diff
@@ -4,9 +4,9 @@
 from django_comment_client.permissions import has_permission
 %>
 
-<div class="discussion-module" data-discussion-id="${discussion_id | h}">
+<div class="discussion-module" data-discussion-id="${discussion_id | h}" data-user-create-comment="${can_create_comment}" data-user-create-subcomment="${can_create_subcomment}">
```

Comment:
Because in the Python code, it's clearer that the context is the user. In HTML, the context isn't clear, so I wanted to make it clearer.